src/types/Transaction.ts

```diff
@@ -69,23 +87,49 @@ export class Transaction {
             case 1: {
                 return this.getGenesisOperations();
             }
-            case 2: {
-                return this.getPaymentOperations();
-            }
+            // case 2: {
```

Comment:
Any specific reason this is commented right now? If it is not available in LTO, simply delete it together with the getPaymentOperations function

Reply:
removed